src/iter.d.ts

```diff
@@ -0,0 +1,2 @@
+import { iter } from './index';
+export = iter;
```

Comment:
the only change here is importing from ./index instead of ../index. It still broke rename detection though...

Reply:
Oops, I had forgotten to do this for the async typedefs too.